kroto-plus-coroutines/src/test/kotlin/com/github/marcoferrer/krotoplus/coroutines/client/ClientCallBidiStreamingTests.kt

```diff
@@ -208,7 +208,7 @@ class ClientCallBidiStreamingTests {
             }
         }
 
-        verify(exactly = 0) { rpcSpy.call.cancel(any(), any()) }
+        verify(exactly = 1) { rpcSpy.call.cancel("Cancelled by client with StreamObserver.onError()", any()) }
```

Comment:
This needed to be change since when an error is received from the server it automatically closes the request channel.